engine/language_client_codegen/src/typescript/templates/partial_types.ts.j2

```diff
@@ -1,6 +1,7 @@
-import { Image } from "@boundaryml/baml"
-
-import * as types from "./types"
+import type { Image, Audio } from "@boundaryml/baml"
+import type { Checked, Check } from "./types"
+import type { {% for t in types %} {{ t }}{% if !loop.last %}, {% endif %}{% endfor %} } from "./types"
```

Comment:
I think this is a name collision, that users can't use "Check" for their own type name - or is that a decision we already made? (Same also applies to "Image" and "Audio", but I think that one is less egregious.)

cc @imalsogreg

Reply:
That's a good point. We have been doing that in other places it seems like. I'd say let's punt this to a new PR